docs/tracking-javascript-guide.md

```diff
@@ -693,7 +693,65 @@ _paq.push(['trackPageView']);
 
 ## Multiple Piwik trackers
 
-It is possible to track a page using multiple Piwik trackers that point to the same or different Piwik servers. To improve page loading time, you can load `piwik.js` once. Each call to `Piwik.getTracker()` returns a unique Piwik Tracker object (instance) which can be configured.
+By default, the Piwik JavaScript Tracking code collects your analytics data into one Piwik server. The Piwik service URL is specified in your JavaScript Tracking code (for example: `var u="//piwik.example.org";`). In some cases, you may want to track your analytics data into more than just one Piwik server. 
+
+In this section, we explain how you can configure the JavaScript tracking code in order to collect all of your website analytics data into two or more different Piwik servers. It is possible to track your analytics data either into a different website ID on the same server, or record a copy of your data into another Piwik server altogether. 
```

Comment:
"In this section, we explain how you can configure the JavaScript tracking code in order to collect all of your website analytics data into two or more different Piwik servers. "

=> It's actually not only explaining the case where we track the data into different servers but also into multiple sites at once (which is explained again in the first part of the next sentence again). I think we could actually remove the whole paragraph.